apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -396,5 +401,80 @@ export function createBookingPageFixture(page: Page) {
       await scheduleSuccessfullyPage.waitFor({ state: "visible" });
       await expect(scheduleSuccessfullyPage).toBeVisible();
     },
+    selectTimezoneInSettings: async (name: string, settingsPage: Page) => {
+      await settingsPage.locator("#timezone svg").click();
+      name.includes(" ") && (name = name.replace(" ", "_"));
+      await settingsPage.getByTestId(`select-option-${name}`).click();
+      await settingsPage.getByRole("button", { name: "Update" }).click();
+      await settingsPage.getByRole("button", { name: "Don't update" }).click();
+    },
+    assertRightTimezone: async (
+      timezone: string,
+      bookingSchedulingPage: Page,
+      options?: { inConfirmationPage: boolean; shouldConfirmBooking?: boolean }
+    ) => {
+      if (options?.inConfirmationPage) {
+        await expect(bookingSchedulingPage.getByText("America/New_York")).toBeVisible();
+        if (options.shouldConfirmBooking) {
+          await bookingSchedulingPage.getByTestId("confirm-book-button").click();
+          await expect(bookingSchedulingPage.getByTestId("cancel")).toBeVisible();
+        }
+      } else {
+        await expect(
+          bookingSchedulingPage
+            .locator("div")
+            .filter({ hasText: `${timezone}` })
+            .nth(1)
+        ).toBeVisible();
+      }
+    },
+    checkUpdateTimezone: async () => {
+      if (await page.getByRole("button", { name: "Update timezone" }).isVisible()) {
+        page.getByRole("button", { name: "Update timezone" }).click();
+      }
+    },
+    createRegularEventType: async (name: string) => {
+      await page.getByTestId("new-event-type").click();
+      await page.getByPlaceholder("Quick Chat").click();
```

Comment:
These tests will start to fail if we simply change the placeholder. Let’s make them less brittle by using element IDs.